documentation/docs/extensibility.md

```diff
@@ -67,6 +67,11 @@ return this
     Please note that the `Init` stage also checks out your current repository including your extensions.<br />
     Therefore, it is not possible to use extensions on this stage.
 
+!!! note "Disable Extensions Execution"
+    By default, there is a possibility for extensions to get executed. In case of disabling it, please ensure to set `env.PIPER_DISABLE_EXTENSIONS=true`.
```

Comment:
The `env` is the environment object in Groovy, I suggest to make the description more general: 

```suggestion
    By default, there is a possibility for extensions to get executed. In case of disabling it, please ensure to set `PIPER_DISABLE_EXTENSIONS` to `true`.
```